### PR TITLE
Added examples/sec hook and loss summary

### PR DESCRIPTION
**Pull request by tfboyd**
Adds hook that prints examples per second and a loss hook to view loss more often as that aspect of estimator is hard coded to only show loss every 100 steps, which is painful for CPU.

### CODE REVIEW

File: tutorials/image/cifar10_estimator/cifar10_main.py [hunk 1]
```diff
@@ -366,21 +433,28 @@ def main(unused_argv):
...
   sess_config.gpu_options.force_gpu_compatible = FLAGS.force_gpu_compatible
+  # Turns on XLA, which needs to be compiled into TensorFlow from source.
+  if FLAGS.xla:
+    sess_config.graph_options.optimizer_options.global_jit_level = tf.OptimizerOptions.ON_1
   config = config.replace(session_config=sess_config)
 
   classifier = tf.estimator.Estimator(
       model_fn=_resnet_model_fn, model_dir=FLAGS.model_dir, config=config)
 
-  tensors_to_log = {'learning_rate': 'learning_rate'}
```
Comment (mari-linhares) [inline tutorials/image/cifar10_estimator/cifar10_main.py]: Shouldn't you keep the learning_rate as well?

File: tutorials/image/cifar10_estimator/cifar10_main.py [hunk 2]
```diff
@@ -366,21 +433,24 @@ def main(unused_argv):
   sess_config.intra_op_parallelism_threads = FLAGS.num_intra_threads
   sess_config.inter_op_parallelism_threads = FLAGS.num_inter_threads
   sess_config.gpu_options.force_gpu_compatible = FLAGS.force_gpu_compatible
-  config = config.replace(session_config=sess_config)
```
Comment (mari-linhares) [inline tutorials/image/cifar10_estimator/cifar10_main.py]: Why this was removed?
